src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+        Path reportPath = Paths.get(toFileName);
+        try {
+            if (Files.exists(reportPath)) {
+                Files.delete(reportPath);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't remove file", e);
+        }
```

Comment:
You don't need these logic
FileReader has constructor that accept String fileName

```suggestion
       public void getStatistic(String fromFileName, String toFileName) {
        String [] data = readFromFile(fromFileName);
        String reportData = createReport(data);
        writeToFile(toFileName, reportData);
    }
```